### DISCUSSION TITLE
🌌 Get a Galaxy Brain For answering 🧠

### DISCUSSION THREAD

**Discussion by the1Riddle** [the1Riddle/binary_trees#13] [Q&A] [answered]
get two of your answers marked as an answer to get a galaxy brain badge

**Comment by habibaadam** [comment]
I love Python and JS😹🖤

**Reply by the1Riddle** [reply]
that's awesome, I'm passionate about C and C# .

**Comment by john-otienoh** [comment, marked as answer]
I'm passionate about JavaScript

**Comment by john-otienoh** [comment]
I'm passionate about python
